Dashboard/app/js/lib/views/maps/map-views-common.js

```diff
@@ -814,22 +822,30 @@ FLOW.NavMapsView = FLOW.View.extend({
     questionGroupsAjaxObject['url'] = '/rest/question_groups?surveyId='+formId;
     questionGroupsAjaxObject['data'] = '';
 
-    FLOW.ajaxCall(function(questionGroupsResponse){
+    FLOW.ajaxCall(function(questionGroupsResponse, qgObj){
```

Comment:
@muloem initially added in case we needed to pass a variable via the callback but currently unused so removing